__tests__/app.test.js

```diff
@@ -79,6 +72,45 @@ describe("/api", () => {
   });
 });
 
+describe("/api/reviews/:review_id", () => {
+  test("GET - status: 200 - returns an object regarding to the review id", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then(({ body }) => {
+        const result = {
+          review_id: 1,
+          title: "Agricola",
+          designer: "Uwe Rosenberg",
+          owner: "mallionaire",
+          review_img_url:
+            "https://images.pexels.com/photos/974314/pexels-photo-974314.jpeg?w=700&h=700",
+          review_body: "Farmyard fun!",
+          category: "euro game",
+          created_at: "2021-01-18T10:00:20.514Z",
+          votes: 1,
+        };
+        expect(body.reviewId[0]).toEqual(result);
+      });
+  });
+  test("GET - status: 404 - returns a message when review id is invalid integer", () => {
```

Comment:
You just need to update the description to reflect what is actually being tested here. You are testing for what happens for a valid but non-existent review ID  👍